test/unit/backend/random_data_generator_test.rb

```diff
@@ -1,7 +1,7 @@
+# frozen_string_literal: true
```

Comment:
After this line, there should be an empty line :smile: 
But I wouldn't request you to push a new commit only for that :smile: :+1: